pkg/cgroup/path_handler.go

```diff
@@ -26,12 +26,23 @@ import (
 )
 
 func SearchByContainerID(topFolder, containerID string) string {
-	found := filepath.Walk(topFolder,
-		func(path string, info os.FileInfo, err error) error {
-			if path == topFolder {
-				return nil
+	found := filepath.WalkDir(topFolder,
+		func(path string, info os.DirEntry, err error) error {
+			if strings.Contains(info.Name(), containerID) {
```

Comment:
to avoid crash on mac, test `info` first
to avoid crash on mac, test `info` first:
```go
if info != nil && strings.Contains(info.Name(), containerID)
```

Reply:
@rootfs Thank you. Changes applied.